bin/runtime-common/src/messages.rs

```diff
@@ -88,10 +91,14 @@ pub trait ChainWithMessageLanes {
 	type Weight: From<frame_support::weights::Weight>;
 	/// Type of balances that is used on the chain.
 	type Balance: CheckedAdd + CheckedDiv + CheckedMul + PartialOrd + From<u32> + Copy;
+
+	/// Instance of the message-lane pallet.
+	type MessageLaneInstance: Instance;
```

Comment:
So this is where you mentioned that we'd need to have `MessageLane<Instance42>` if our bridged chain has a bridge pallet with `Instance42` used for connecting with us?

Reply:
Not exactly here. Instances are exported from `pallet_message_lane`, so they're available in all runtimes => you won't have any problems here. But when you will call any function from `pallet_message_lane::storage_keys` module with runtime that is not implementing `pallet_message_lane::Trait<Instance42>`, you will see compilation issue.